### DISCUSSION TITLE
设置重启时间为默认0 .3      ，版本升级如何更新

### DISCUSSION THREAD

**Discussion by sscjjo** [nikkinikki-org/OpenWrt-nikki#368] [Q&A]
设置重启时间为默认的0 3   但是看日志感觉后台时间和现实时间不是同步的

另外如1.40在线升到1.5该怎么升级

**Comment by morytyann** [comment]
系统时区设置了吗？

**Comment by morytyann** [comment]
升级看你怎么安装的，从feed.sh安装的直接在软件源升，通过install.sh脚本安装的再运行一次脚本就可以升。
